packages/file-format-ts/scripts/generate.ts

```diff
@@ -18,8 +14,8 @@ type SchemaMap = {
   [key: string]: JSONSchema7
 }
 
-const generate = (version: string, schemas: any) => {
-  const outFile = `${version}-types.ts`
+const generate = (schemas: any) => {
```

Comment:
It's likely that we can type better this any, since the schemas come from the `sketch-file-format` repo.

Let me try it.

Reply:
Well, it's now in this same repo. I'm trying in any case to get better typing for this `any`.